metricflow/query/query_parser.py

```diff
@@ -6,7 +6,11 @@
 from dataclasses import dataclass
 from typing import List, Optional, Tuple, Dict, Sequence
 
-import fuzzywuzzy.fuzz
+import warnings
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=UserWarning)
```

Comment:
Is it possible to filter this specific warning? If not that's fine, it's a nice-to-have. We don't maintain fuzzywuzzy.

Reply:
I saw a warning capture that let you filter on the error message but it was only available as a method decorator (and I also now can no longer find an example of the thing I saw previously).

It seems like warnings.filterwarnings can only filter on the class of the warning, rather than the content of it.